src/Shared/FileMatcher.cs

```diff
@@ -1869,20 +1864,17 @@ out matchResult.isLegalFileSpec
             string fileToMatch,
             Regex fileSpecRegex,
             out bool isMatch,
-            out string fixedDirectoryPart,
             out string wildcardDirectoryPart,
             out string filenamePart)
         {
             Match match = fileSpecRegex.Match(fileToMatch);
 
             isMatch = match.Success;
-            fixedDirectoryPart = string.Empty;
             wildcardDirectoryPart = String.Empty;
-            filenamePart = string.Empty;
```

Comment:
Don't we generally prefer lower-case `string`, since it always refers to the right type while `String` might be some other type?

Reply:
Fixed, thank you.